client/state/comments/README.md

```diff
@@ -0,0 +1,29 @@
+Comments
+========
+
+## This store has the following structure:
+
+```
+comments = {
+	items: Map<CommentTargetId, Tree>,
```

Comment:
@blowery 
Addressed with [this](https://github.com/samuelclemens/wp-calypso/commit/866e9a3fbfd4303dddb9665410a5882d5506ed54)
Not sure if it's the solution for that.